sub_scripts/lxc_build.sh

```diff
@@ -88,7 +88,13 @@ then	# Si le conteneur existe déjà
 fi
 
 echo -e "\e[1m> Création d'une machine debian $DISTRIB minimaliste.\e[0m" | tee -a "$LOG_BUILD_LXC"
-sudo lxc-create -n $LXC_NAME -t debian -- -r $DISTRIB >> "$LOG_BUILD_LXC" 2>&1
+uname=$(uname -m)
+if [[ $uname =~ aarch64 ]]
+then
+        sudo lxc-create -n $LXC_NAME -t debian -- -r $DISTRIB  --arch=arm64 >> "$LOG_BUILD_LXC" 2>&1
```

Comment:
Then here, use a variable to add `--arch=`, so you will not duplicate the entire line.

Reply:
I changed it and tested it. It works